### DISCUSSION TITLE
Safari only: `x-model` does not detect an "autocompleted" input change

### DISCUSSION THREAD

**Discussion by calebrunion** [alpinejs/alpine#1501] [7. Bugs (archived)]
I use `x-model` to disable a form's submit button until its related fields have values.

The following code _used_ to work in V2 across modern browsers. But in V3, it breaks when you autocomplete an input in Safari (macOS and iOS). However, it continues to work as expected in Chromium & Firefox.
```
<form x-data="{ email: '' }">
    <input type="text" name="email" x-model="email" autocomplete="email">
    <button x-bind:disabled="!email" type="submit">Sign Up</button>
</form>
```
Note: This bug is only for the **autocomplete** case; `x-model` will detect change when you simply type in the input.

**Comment by MuzafferDede** [comment]
- #1969 should fix this
